FGEGraphics/UISystem/UIElement.cs

```diff
@@ -617,9 +668,15 @@ public virtual void Destroy()
     /// <summary>Returns debug text to add to <see cref="ViewUI2D.InternalData.DebugInfo"/>.</summary>
     public virtual List<string> GetDebugInfo()
     {
-        string type = $"^t^0^h^5^u{GetType()}";
-        string pos = $"^r^t^0^h^o^e^7Position: ^3({X}, {Y}) ^&| ^7Dimensions: ^3({Width}w, {Height}h) ^&| ^7Rotation: ^3{LastAbsoluteRotation}";
-        string state = $"^7Enabled: ^{(Enabled ? "2" : "1")}{Enabled} ^&| ^7Hovered: ^{(Hovered ? "2" : "1")}{Hovered} ^&| ^7Pressed: ^{(Pressed ? "2" : "1")}{Pressed}";
-        return [type, pos, state];
+        List<string> info = [];
+        info.Add($"^t^0^h^5^u{GetType()}");
+        info.Add($"^r^t^0^h^o^e^7Position: ^3({X}, {Y}) ^&| ^7Dimensions: ^3({Width}w, {Height}h) ^&| ^7Rotation: ^3{LastAbsoluteRotation}");
+        info.Add($"^7Enabled: ^{(Enabled ? "2" : "1")}{Enabled} ^&| ^7Hovered: ^{(Hovered ? "2" : "1")}{Hovered} ^&| ^7Pressed: ^{(Pressed ? "2" : "1")}{Pressed}");
+        if (ElementInternal.Styles.Count > 0)
+        {
+            List<string> styleNames = ElementInternal.Styles.Select(style => style.Name is string name ? $"^{(style == ElementInternal.CurrentStyle ? "3" : "7")}{name}" : "^1unnamed").ToList();
```

Comment:
`style.Name is string name` it sure is a string yes. is this meant to be a null check?

Reply:
also `ToList` -- is VS not highlighting that to autoreplace with `[.. x]` syntax?

Reply:
> `style.Name is string name` it sure is a string yes. is this meant to be a null check?

I'm used to nullables being enabled wherein this is how you use nullable objects that you null check for. Can just switch it out for a `is not null` if that's cleaner.

Reply:
Yes use `is not null`

Even when `is string` _works_ as a null check, it is not _clear_ as a null check - give two relatively equal choices, the one that's clearer about intent is definitely better